vendor/engines/secure_rooms/db/migrate/20170301120659_create_card_reader_and_control_device.rb

```diff
@@ -0,0 +1,19 @@
+class CreateCardReaderAndControlDevice < ActiveRecord::Migration
+
+  def change
+    create_table :secure_rooms_control_devices do |t|
+      t.references :product, null: false
+      t.foreign_key :products
```

Comment:
oy, I continue to underestimate oracle's situation 😵 might as well add it if I'm updating here